ocean_provider/utils/util.py

```diff
@@ -329,7 +329,7 @@ def decode_from_data(data, key, dec_type="list"):
 def service_unavailable(error, context, custom_logger=None):
     text_items = []
     for key, value in context.items():
-        text_items.append(key + "=" + value)
+        text_items.append(key + "=" + str(value))
```

Comment:
Why is this change necessary?

Reply:
Something broke byzantinely. I fixed it :)) Simple as that.